index.html

```diff
@@ -11053,6 +11053,64 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-brailleroledescription">
+			<pdef>aria-brailleroledescription</pdef>
+			<div class="property-description">
+				<p>Defines a human-readable, author-localized Braille description for the <a>role</a> of an <a>element</a>.</p>
```

Comment:
I think this could be better described as: "Defines a human-readable, author-localized short description for the role of an element, which is intended to be converted into Braille."

Reply:
Perhaps "abbreviated role description"

Reply:
Thanks, @cookiecrook, I'll make a new PR to address this.